.github/workflows/e2e-sync-ui.yml

```
@@ -0,0 +1,82 @@
+               - ボタン/リンクのテキスト変更
+               - フォーム要素の構造変更
+            4. 必要に応じてPage Objectのロケータを更新
+            5. テストケースに影響がないか確認
+
+            ## ロケータ優先順位
+            getByRole > getByLabel > getByText > getByTestId
+
+            ## 修正があれば
+            git add と git commit を実行してください。
+            コミットメッセージ: "fix(e2e): UIコンポーネント変更に追従"
+          allowed_tools: "Read,Edit,Write,Bash(git:*),Glob,Grep"
```

actionlint (1.7.9)

input "allowed_tools" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_prefix", "claude_args", "claude_code_oauth_token", "experimental_allowed_domains", "github_token", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_sticky_comment", "use_vertex"

(action)